curriculum/challenges/english/20-upcoming-python/learn-tree-traversal-by-building-a-binary-search-tree/655a1564f3aa8210938cdf68.md

```diff
@@ -0,0 +1,53 @@
+---
+id: 655a1564f3aa8210938cdf68
+title: Step 17
+challengeType: 20
+dashedName: step-17
+---
+
+# --description--
+
+Add another condition using `elif` that checks if the `key` > `node.key`.
+
+# --hints--
+
+You should use the `elif` keyword for the second condition.
+
+```js
+({ test: () => assert.match(code, /elif/) });
+```
+
+Check if the `key` > `node.key`.
+
+```js
+({ test: () => assert.match(code, /key\s*>\s*node\.key/) });
+```
```

Comment:
As for the previous step I would replace these two with:

You should add an `elif` conditional statement to check if `key > node.key`.

```js
({ test: () => assert.match(e.code.original["main.py"], /^(\s+)if\s+node\s+is\s+None\s*:.*?\1elif\s+key\s*>\s*node\.key\s*:/ms) })
```